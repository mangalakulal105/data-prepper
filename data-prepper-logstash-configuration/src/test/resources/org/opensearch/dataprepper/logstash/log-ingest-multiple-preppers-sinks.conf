input {
    http {
    }
}
filter {
    grok {
        match => {"log" => "%{COMBINEDAPACHELOG}"}
    }
    grok {
        pattern_definitions => {
            "CUSTOMYEAR" => "(?>\d\d){1,2}"
            "CUSTOMMINUTE" => "(?:[0-5][0-9])"
        }
        match => {"otherlog" => "%{DATA:message} %{CUSTOMYEAR:year} %{CUSTOMMINUTE:minute}"}
    }
}
output {
    elasticsearch {
        hosts => ["https://localhost:9200"]
        user => admin
        password => mypassword
        index => "logingest"
    }
    amazon_es {
        hosts => ["fakedomain.us-east-1.es.amazonaws.com"]
        region => "us-east-1"
        index => "logingest"
    }
}