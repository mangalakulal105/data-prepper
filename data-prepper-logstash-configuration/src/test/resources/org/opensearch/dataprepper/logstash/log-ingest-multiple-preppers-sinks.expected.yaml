logstash-converted-pipeline:
  source:
    http:
      max_connection_count: 500
      request_timeout: 10000
  processor:
    - grok:
        match:
          log:
            - "%{COMBINEDAPACHELOG}"
    - grok:
        match:
          otherlog:
            - "%{DATA:message} %{CUSTOMYEAR:year} %{CUSTOMMINUTE:minute}"
        pattern_definitions:
          CUSTOMMINUTE: "(?:[0-5][0-9])"
          CUSTOMYEAR: "(?>\\d\\d){1,2}"
  sink:
    - opensearch:
        hosts:
          - "https://localhost:9200"
        username: "admin"
        password: "mypassword"
        index: "logingest"
    - opensearch:
        aws_sigv4: true
        insecure: false
        hosts:
          - "fakedomain.us-east-1.es.amazonaws.com"
        aws_region: "us-east-1"
        index: "logingest"
